flash/text/classification/data.py

```diff
@@ -278,6 +278,49 @@ def __setstate__(self, state):
         self.tokenizer = AutoTokenizer.from_pretrained(self.backbone, use_fast=True)
 
 
+class TextListDataSource(TextDataSource):
+    def load_data(
+        self,
+        data: Tuple[List[str], List[Any]],
+        dataset: Optional[Any] = None,
+        columns: Union[List[str], Tuple[str]] = ("input_ids", "attention_mask", "labels"),
+    ) -> Union[Sequence[Mapping[str, Any]]]:
+        input, target = data
+        hf_dataset = Dataset.from_dict({"input": input, "labels": target})
+
+        if not self.predicting:
+            dataset.multi_label = False
```

Comment:
Would you mind adding support for multi label ?

Reply:
No problem :) What should `targets` look like? A list of `num_samples` lists, each has `num_classes` items? Or a list of `num_classes` lists, each has `num_samples` items?